src/assets/Generator.Shared/CodeGenClientAttribute.cs

```diff
@@ -1,13 +1,18 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+#nullable enable
+
 using System;
 
 namespace Azure.Core
 {
     [AttributeUsage(AttributeTargets.Class)]
     internal class CodeGenClientAttribute : CodeGenTypeAttribute
     {
+        public Type? ParentClient { get; set; }
+        public bool ForcePublicConstructors { get; set; }
```

Comment:
Remind me what ForcePublicConstructors means?

Reply:
We generate public constructors for the client no matter if it is a subclient or not.

Reply:
Could we add a comment to document this?

Reply:
Removed `ForcePublicConstructors` per https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-subclient-no-construct